DietPi-Software | FreshRSS

Joulinar opened the pull request:
DietPi-Software | FreshRSS - GitHub branch has been renamed from Master to Edge

Joulinar committed:
v8.10

DietPi-Software | FreshRSS - GitHub branch has been renamed from Master to Edge

MichaIng reviewed (changes requested):
We should pull the latest stable release then: https://api.github.com/repos/FreshRSS/FreshRSS/releases/latest
If I'm not mistaken, `master` previously matched the latest release while edge existed before as development branch.

MichaIng committed:
v8.10

- DietPi-Software | FreshRSS: Resolved an issue where the install failed because of changed branches on the GitHub repository. Many thanks to @baptiste313 for reporting this issue: https://github.com/MichaIng/DietPi/issues/5767

MichaIng committed:
v8.10

- DietPi-Software | FreshRSS: Apply strong random database password on fresh installs and add further CLI steps beneficial for reinstalls

MichaIng committed:
v8.10

- DietPi-Software | Syntax
